surefire-providers/common-junit48/src/test/java/org/apache/maven/surefire/common/junit48/GroupMatcherCategoryFilterPreJUnit412Test.java

```diff
@@ -0,0 +1,65 @@
+package org.apache.maven.surefire.common.junit48;
```

Comment:
Pls put a new line between package and the comment with license. The checkstyle may fail right now.